test/runtests.jl

```diff
@@ -34,3 +34,10 @@ println("Testing miscellaneous functionality...")
 tic()
 include("MiscTest.jl")
 println("done (took $(toq()) seconds).")
+
+if Base.JLOptions().opt_level >= 3 && VERSION >= v"0.5"
```

Comment:
@KristofferC Do you know what I should add to this conditional to disable these tests for users who won't get SIMD in the LLVM IR?

Also is there any point to me getting this to run on Travis, since Travis just downloads Julia binaries, or are these tests doomed to be local only?